src/components/info-container.tsx

```diff
@@ -24,41 +28,51 @@ const styles = StyleSheet.create({
 	},
 });
 
-interface Props {
-	curLocation: LocationObject | null;
-	mobileNetCode: string | null;
-	routeCoordinates: Array<LatLng>;
-	routeId: string;
-}
-
 /**
  * Info container component to show information primarily for debugging
  * purposes. Some of the info should be deleted later, but perhaps keep the coordinates?
  */
-const InfoContainer = ({
-	curLocation,
-	mobileNetCode,
-	routeCoordinates,
-	routeId,
-}: Props): JSX.Element => {
+const InfoContainer = (): JSX.Element => {
+	const [curLocation] = useState<LocationObject | null>(null);
+	const [mobileNetCode] = useState<string | null>(null);
+	const waypoints = useTypedSelector((state) => state.waypoints);
+	const routeId = useTypedSelector((state) => state.route.routeId);
+
+	/**
+	 * These are not updating currently. The problem is that the params this container
+	 * at this container can not be updated unless the MapView component is active.
+	 * Therefore changing to different views will cause memory leak warning.
+	 * At the moment I don't see much reasons to fix this component.
+	 */
+	// useEffect(() => {
```

Comment:
I'm not totally sure what you mean here but if the effect is not needed, I think it should be removed from the code completely instead of being just commented out — version control is there if we need the effect code in the future!

Reply:
Removed